### PR TITLE
Update dependencies

### PR DESCRIPTION
**Pull request by agriffard**
BootstrapSlider: 10.6.2
CodeMirror: 5.48.4
jQuery: 3.4.1
jsplumb: 2.11.2
graphiql: 0.14.2
graphql: 14.5.4
graphiql-explorer: 0.4.4
react: 16.9.0

+ Readme and ResourceManifest (versions, CDN, hash)

Trumbowyg will be updated in another PR.

/cc @jptissot @deanmarcussen

### CODE REVIEW

**Comment by jptissot** [comment]
LGTM.

Some resources are not being tracked by package.json and are only used in views. Such as vuejs. Should we consolidate all there ? I would be willing to give it a look if you think it's useful.

**Comment by deanmarcussen** [comment]
Nice one, @agriffard that's a bit of work.

how does this make the npm security vulnerabilities listing look? better I imagine

I have a vague memory that the graphiql explorer is built with webpack, and doesn't have a hook to the gulp pipeline. I think the command to build is `npm run-script build`

/cc @jrestall 

re: all the manifest / hashes / cdn's etc looks great

@jptissot I think vue is referenced straight from a cdn, no local copy. But there are a bunch of consolidations and such with other scripts that could definitely happen. Sebastien has suggested that at some point we could find another home for admin related resources, to separate them from theme related resources, and save some of the confusion, which might make it all a bit clearer. 
For me the idea of an `AdminResources` module makes good sense, but @sebastienros might have a much better idea.

**Comment by jptissot** [comment]
@deanmarcussen I think most of npm vulnerabilities come from librairies referenced by gulp and not runtime packages.

**Comment by jrestall** [comment]
> I have a vague memory that the graphiql explorer is built with webpack, and doesn't have a hook to the gulp pipeline. I think the command to build is `npm run-script build`
> 
> /cc @jrestall

Correct, if you are updating the graphiql dependencies you'll need to run the above and commit the updated graphiql-orchard.js file.

**Comment by sebastienros** [comment]
Can you rebuild the assets, I must have broken them by merging another PR

**Comment by deanmarcussen** [comment]
@agriffard Could you merge dev and do the rebuild of assets?

**Comment by agriffard** [comment]
Ok, dev merged and assets rebuilt.

**Comment by jptissot** [comment]
@agriffard which npm / node version are you running ?

File: package-lock.json [hunk 1]
```diff
@@ -5018,79 +4955,84 @@
...
         "bootstrap": "4.3.1",
         "bootstrap-scss": "4.3.1",
-        "bootstrap-slider": "10.4.1",
-        "codemirror": "5.42.0",
-        "jquery": "3.3.1",
-        "popper.js": "1.14.7",
+        "bootstrap-slider": "10.6.2",
+        "codemirror": "5.48.4",
+        "jquery": "3.4.1",
+        "popper.js": "1.15.0",
```
Comment (jptissot) [inline package-lock.json]: I am not sure this is what we want.

**Comment by agriffard** [comment]
nodejs-lts v10.16.3

**Comment by agriffard** [comment]
Upgraded npm to 6.11.3

**Comment by jptissot** [comment]
Do you delete node_modules and run a npm run rebuild from the root of the project? I was getting weird inconsistent behavior yesterday

**Comment by agriffard** [comment]
When I delete node_modulesand rebuild, there is an error to build because bootstrap-slider is not found.
I remember I retirieved it manually to make it work the last time.
The package-lock file seem to be changed also.

**Comment by sebastienros** [comment]
Should I merge this? Are the built assets up to date or will it regress recent scripting changes?

**Comment by agriffard** [comment]
Yes, I would like this PR too be merged so I can use the latest versions in my PR about content items filter and add other dependencies like bootstrap-select.

**Pull request merged**